stubs/common/Pivot.stub

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Illuminate\Database\Eloquent\Relations;
+
+class Pivot
+{
+}
```

Comment:
can you please update the MorphToMany stub in accordance with the framework?

```suggestion
class Pivot extends Model {}
class MorphPivot extends Pivot {}
```

Reply:
There isn't a MorphToMany stub so it should already be using the one from the framework.

Reply:
Ah, good deal 👌

Reply:
Oh, you want it so that it applies even before they release the next version, got it.

Reply:
It should be fine since the BelongsToMany uses defaults now, the MorphToMany will just become smarter when the release happens.


I hate maintaining duplicate stubs lol